src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,59 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+public class Intake extends SubsystemBase {
+    private TalonSRX motor = new TalonSRX(Ports.Intake.MOTOR);
+    private Solenoid solenoidR = new Solenoid(Ports.Intake.SOLENOID_RIGHT);
+    private Solenoid solenoidL = new Solenoid(Ports.Intake.SOLENOID_LEFT);
+    private UnitModel unitModel = new UnitModel(Constants.Intake.TICK_PER_METER);
+
+    /**
+     * sets the motor inverted
+     */
+    public Intake() {
+        motor.setInverted(Ports.Intake.IS_INVERTED);

```

Comment:
Add sensor phase

Reply:
Configure the encoder

Reply:
@Adam14boat does the intake have an encoder?

Reply:
He uses a get velocity method, you are probably right and it is unnecessary